### PR TITLE
Add toggle for strawpoll options to be added by delegates

### PR DESCRIPTION
**Pull request by kumar-ish**
As per #84, we want the ability for the conference organiser to toggle an option that lets delegates add their options to strawpolls (but does not let them delete them). This pull request implements this. 

Demo below:

output

### CODE REVIEW

File: src/components/Strawpoll.tsx [hunk 1]
```diff
@@ -319,41 +322,54 @@ export default function Strawpoll(props: StrawpollProps) {
...
+              </Button.Group>
+            </List.Item>
+            <List.Item >
+              <Checkbox
+                label="Delegates can add options"
+                toggle
+                checked={strawpoll ? (strawpoll.addingOptionsIsPublic || false) : false}
+                onClick={checkboxHandler<StrawpollData>(strawpollFref, 'addingOptionsIsPublic')}
+              />
+
```
Comment (kumar-ish) [inline src/components/Strawpoll.tsx:370]:
```suggestion
```

File: database.rules.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
           "$sid": {
             "options": {
               "$oid": {
+                ".write": "newData.exists() && data.parent().parent().child('addingOptionsIsPublic').val() === true",
```
Comment (MaxwellBo) [inline database.rules.json:36]: nit: can this be `optionsArePublic`? Seems more inline with the convention of `<type>(Is|Are)Public`

Comment (MaxwellBo) [reply database.rules.json:36]: - [ ] TODO: @MaxwellBo Update the `database.rules.json` in the Firebase console after this change is merged

File: src/components/Strawpoll.tsx [hunk 3]
```diff
@@ -319,41 +322,53 @@ export default function Strawpoll(props: StrawpollProps) {
...
+                      onClick={()=> setOpen(true)}
+                    >
+                      <Icon name="delete" />Delete strawpoll?
+                    </Button>
+                  }
+                />
+              </Button.Group>
+            </List.Item>
+            <List.Item >
+              <Checkbox
```
Comment (MaxwellBo) [inline src/components/Strawpoll.tsx:364]: Should this have a `loading={!strawpoll}`?

File: src/components/Strawpoll.tsx [hunk 4]
```diff
@@ -319,41 +322,53 @@ export default function Strawpoll(props: StrawpollProps) {
...
+                label="Delegates can add options"
+                toggle
+                checked={strawpoll ? (strawpoll.addingOptionsIsPublic || false) : false}
```
Comment (MaxwellBo) [inline src/components/Strawpoll.tsx:367]: I think this could be simplified to `checked={strawpoll?.addingOptionsIsPublic}`

**Review by MaxwellBo — CHANGES_REQUESTED** [review]

**Pull request merged**